themes/default/views/admin/orders/show.blade.php

```diff
@@ -2,249 +2,282 @@
-                                                    </div>
-                                                    <div class="flex flex-col mt-4">
-                                                        <span class="font-bold">{{ __('Expiry Date') }}:</span>
-                                                        <input class="form-input" type="date" name="expiry_date"
-                                                            value="{{ $product->expiry_date }}"
-                                                            placeholder="{{ __('Expiry Date') }}" />
-                                                    </div>
-                                                    <div class="flex flex-row justify-end mt-4">
-                                                        <button class="form-submit" type="submit">
-                                                            {{ __('Change') }}
-                                                        </button>
-                                                    </div>
-                                                </form>
-                                            </div>
-                                        </div>
+                                <div id="tooltip-animation2" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                    {{__('Change price/quantity')}}
+                                    <div class="tooltip-arrow" data-popper-arrow></div>
+                                </div>
+                            </div>
+
+                            @if(!$product->link)
+                                <div>
+                                    <form method="POST" action="{{ route('admin.orders.create', $order->id) }}">
+                                        @csrf
+                                        <button class="button button-success" type="submit" data-tooltip-target="tooltip-animation5">
+                                            <i class="ri-add-circle-line"></i>
+                                        </button>
+                                    </form>
+                                    <div id="tooltip-animation5" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                        {{__('Create product')}}
+                                        <div class="tooltip-arrow" data-popper-arrow></div>
                                     </div>
                                 </div>
-                            </span>
-                        </div>
+                            @endif
+
+                            @if ($order->products[0]->status == 'pending')
```

Comment:
Use `$product->status` here instead of `$order->products[0]->status`

Reply:
Corrected, thanks